recipe/build.sh

```diff
@@ -24,3 +26,10 @@ cmake ${CMAKE_ARGS} \
 
 cmake --build ./build
 cmake --build ./build --target install
+
+# TODO: currently debug symbols are included in Release build by design.
+#   Should we create a split package with debug symbols, like ddeb?
+#   https://github.com/intel/intel-graphics-compiler/issues/341
+find $PREFIX/lib | grep $SHLIB_EXT.$PKG_VERSION | xargs $STRIP
```

Comment:
You might accidentally strip other libraries in $PREFIX/lib leading to corruption of ocnda pkgs in the machine you ran this code.

Reply:
Yeah, that's a good point! Do you know any ways without listing libraries individually? I was syncing of installing in separate folder, do the strip there and then rsync on $PREFIX. What do you think?